src/factions/AdeptaSororitas/models.json

```diff
@@ -0,0 +1,465 @@
+[
```

Comment:
Now that you have the jsonnet file, you need to  remove this one, they'll conflict in the build and I don't think my build script is smart enough to crash

Reply:
👍